### PR TITLE
[Events] KubeCon China: Adding Xin Huang

### PR DESCRIPTION
**Pull request by leecalcote**
**Description**

Adding Xin Huang to SMP Project Office Hours

### CODE REVIEW

**Comment by leecalcote** [comment]
@gyohuangxin for your review...

**Comment by l5io** [comment]
🚀 Preview for commit 3649c642df275607ba533aa00dcad65e65e69c1d at: https://61a7f13caa6ed000586465a0--layer5ng.netlify.app

**Review by gyohuangxin — APPROVED** [review]
LGTM, thanks.

File: src/collections/events/kubecon-china-2021/index.mdx [hunk 1]
```diff
@@ -95,7 +95,12 @@ Speakers:
 
 Speakers:
 <ul>
-<li>All Service Mesh Performance project maintainers</li>
+<li><Link to="/community/members/aisuko-li">Sunku Ranganath</Link></li>
```
Comment (gyohuangxin) [inline src/collections/events/kubecon-china-2021/index.mdx:98]: A small mistake, Sunku's profile was linked to Aisuko, and the same mistake in Chinese version.

Comment (leecalcote) [reply src/collections/events/kubecon-china-2021/index.mdx:98]: Uh-oh. Thanks for pointing this out.

**Pull request merged**